packages/camera/camera_web/lib/src/camera_web.dart

```diff
@@ -193,11 +193,10 @@ class CameraPlugin extends CameraPlatform {
   }
 
   @override
-  Future<int> createCamera(
```

Comment:
Please, do not remove `createCamera`, this is still a valid method in the platform_interface and not deprecated.

Instead, can you reimplement it by delegating to the new `createCameraWithSettings`?

Something like:

```dart
Future<int> createCamera( ...old_params...) {
  return createCameraWithSettings(
    MediaSettings(...created using old_params...)
  );
}
```

Reply:
reimplemented

Reply:
`createCamera` reimplemented